lib/Service/ContactsIntegration.php

```diff
@@ -106,10 +106,12 @@ public function getMatchingRecipient(string $term): array {
 	 * @return false|null|string
 	 */
 	public function getPhoto(string $email) {
-		$result = $this->contactsManager->search($email, ['EMAIL']);
+		$result = $this->contactsManager->search($email, ['EMAIL'], ['escape_like_param' => false]);
 		if (count($result) > 0) {
```

Comment:
You can remove the `if` and just leave the `foreach`. This it will "skip" the loop if the array is empty.